src/Transports/MassTransit.AmazonSqsTransport/Contexts/AmazonSqsReceiveContext.cs

```diff
@@ -128,15 +129,12 @@ async Task RenewMessageVisibility()
                     if (_activeTokenSource.Token.IsCancellationRequested)
                         break;
 
-                    visibilityTimeout = Math.Min(MaxVisibilityTimeout, visibilityTimeout * 2);
-
                     await _clientContext.ChangeMessageVisibility(_receiveSettings.QueueUrl, TransportMessage.ReceiptHandle, visibilityTimeout)

```

Comment:
The new timeout period takes effect from the time you call the ChangeMessageVisibility action. In addition, the new timeout period applies only to the particular receipt of the message. ChangeMessageVisibility doesn't affect the timeout of later receipts of the message or later queues.

For example, if the default timeout for a queue is 60 seconds, 15 seconds have elapsed since you received the message, and you send a ChangeMessageVisibility call with VisibilityTimeout set to 10 seconds, the 10 seconds begin to count from the time that you make the ChangeMessageVisibility call. 

https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html